highfidelity/hifi
# (case 19302) Re-name FBXGeometry to HFMGeometry and do the same for related classes

https://highfidelity.manuscript.com/f/cases/19302/improve-model-loading-tool-chain-project-Phase-1

As part of Phase 1 of the Model Improvement project, this PR re-names FBXGeometry and related classes to use HFM instead of FBX in the names. This is done now rather than later to prevent merge conflicts later.

## TEST PLAN
- Run these asset browser baking tests:
   - https://highfidelity.testrail.net/index.php?/cases/view/17647
   - https://highfidelity.testrail.net/index.php?/cases/view/18116
 - Run these auto tests for verifying model loading:
   - https://github.com/highfidelity/hifi_tests/tree/master/tests/engine/render/lighting
   - https://github.com/highfidelity/hifi_tests/tree/master/tests/engine/render/material
   - https://github.com/highfidelity/hifi_tests/tree/master/tests/engine/render/shadows/primaryCamera/front
   - https://github.com/sabrina-shanman/hifi_tests/tree/gltf_test/tests/content/entity/model/modelReaders
     - Note: These tests are in a different location from the other tests
- Run manual tests for verifying model loading:
  - https://github.com/highfidelity/hifi_tests/tree/master/tests/engine/render/geometry/skinning
  - In this PR, look at the way the avatar walks and make sure it looks normal
  - In this PR, use an emote from the emotes app and make sure it looks normal
  - In this PR and in master, using an avatar that supports facial expressions, run this script in desktop mode: https://cdn.rawgit.com/thoys/hifi-content/346a2ae1e0a062619c7d25c48fe51adf58298ed9/Prototyping/FacialExpressions/facialExpressions.js
    - Press "H" on the keyboard to make the avatar make a face. Compare to master and confirm it looks the same.
